Python/compile.c

```diff
@@ -2491,22 +2470,21 @@ codegen_class(struct compiler *c, stmt_ty s)
         if (!type_params_name) {
             return ERROR;
         }
-        if (compiler_enter_scope(c, type_params_name, COMPILER_SCOPE_ANNOTATIONS,
-                                 (void *)type_params, firstlineno, s->v.ClassDef.name, NULL) == -1) {
-            Py_DECREF(type_params_name);
-            return ERROR;
-        }
+        int ret = compiler_enter_scope(c, type_params_name, COMPILER_SCOPE_ANNOTATIONS,
+                                 (void *)type_params, firstlineno, s->v.ClassDef.name, NULL);
```

Comment:
```suggestion
        int ret = compiler_enter_scope(c, type_params_name, COMPILER_SCOPE_ANNOTATIONS,
                                       (void *)type_params, firstlineno, s->v.ClassDef.name, NULL);
```

I hope it's not exceeding the line length =/